DataHub/Integration/02_ManageAttributeSets.md

```diff
@@ -0,0 +1,308 @@
+
+#### Procedure
+
+*DataHub > Settings > Tab ATTRIBUTE SETS*
+
+![Attribute sets](../../Assets/Screenshots/DataHub/Settings/AttributeSets/AttributeSets.png "[Attribute sets]")
+
+1. Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the bottom right corner.   
+  The *Create attribute set* view is displayed.
+
+  ![Create attribute set](../../Assets/Screenshots/DataHub/Settings/AttributeSets/CreateAttributeSet.png "[Create attribute set]")
+
+2. Enter a name for the attribute set in the *Name* field and, if desired, add an attribute set description in the *Description* field.
+
+
+3. Enter a key for the attribute set in the *Key* field. The key is required for API access and must be system wide unique. An attribute set key must fulfill the following criteria:
+  - valid characters are **a-z** (upper and lower case), **0-9** and the underscore ( **_** )
+  - the key must not start with a number
+  - a double underscore ( **___** ) and a trailing underscore are forbidden
+
+
+  > [Info] In order to facilitate the assignment of sets in the further process, it is recommended to add the prefix **datahubset_** to all attribute sets created in the *DataHub* module.
+
+4. Click the *Entity type* drop-down list and select the appropriate entity type. The list of options may differ depending on the system configuration.
+
+  > Entity types are all classes interacting with the *DataHub* module.
+
+
+5. For the next steps to create an attribute set, follow the appropriate procedure:   
+    - [Create an attribute set without attribute inheritance](#create-an-attribute-set-without-attribute-inheritance)
+    - [Create an attribute set with attribute inheritance](#create-an-attribute-set-with-attribute-inheritance)
+    - [Create an attribute set with attribute copying](#create-an-attribute-set-with-attribute-copying)
+
+
+
+### Create an attribute set without attribute inheritance
+
+When you create an attribute set without inheritance, you have to add all attributes manually to the created attribute set.
+Neither attributes nor configurations are applied from another attribute set.
+In exchange, you can define individually which attributes are required and customize their configuration.
+
+#### Prerequisites
+
+- Additional attributes for the attribute set have been created, see [Create an attribute](./01_ManageAttributs.md#create-an-attribute).
+- The basic data of the attribute set has been completed, see [Define the attribute set basic data](#define-the-attribute-set-basic-data).
+
+#### Procedure
+
+*DataHub > Settings > Tab ATTRIBUTE SETS*
+
+![Create attribute set inheritance](../../Assets/Screenshots/DataHub/Settings/AttributeSets/CreateAttributeSetInheritance.png "[Create attribute set inheritance]")
+
+1. Select the **No inheritance** option in the *Inherit/Copy values from* drop-down list.   
+  The *Inherit configuration* toggle and the *Inherit/Copy values* drop-down list are locked.
+
```

Comment:
Vgl. Groß-/Kleinschreibung im PIM. Ich gehe davon aus, dass es so wie in DataHub durchgängig gemacht werden muss, wie schon besprochen.